src/session/__tests__/sessionReducer-test.js

```diff
@@ -1,9 +1,9 @@
-/* @flow strict-local */
```

Comment:
No, leave this here -- see our [test style guide](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/testing.md#test-style-guide).

I don't totally follow the commit message:

    So that there is no need to dump unwanted data in tests, and thus make
    tests clener.

but I think this might be describing what `exampleData` is the solution for.

Reply:
Thanks for the link.

I was on completely on opposite page 😅